### PR TITLE
Update mozid modal header sizing

### PR DESCRIPTION
**Pull request by alexgibson**
Minor tweak to moz-id modal header styling, as per https://bugzilla.mozilla.org/show_bug.cgi?id=1136937#c27

### CODE REVIEW

File: media/css/base/mozid-modal.less [hunk 1]
```diff
@@ -31,7 +31,8 @@ html.noscroll body {
             &> header {
                 display: block;
                 height: auto;
-                .font-size(60px);
+                .font-size(48px);
+                padding: 0 (@gridGutterWidth * 3);
```
Comment (jpetto) [inline media/css/base/mozid-modal.less]: One addition - would be nice to drop the line height down to 1.3 or so. 1.5 is a bit much when the text wraps.

**Comment by jpetto** [comment]
Minor request to update `line-height` for header while we're modifying the file. Other than that, r+. Merge when ready!

**Comment by jpetto** [comment]
Oh, and :snake:

File: media/css/base/mozid-modal.less (hunk 1 above)
Comment (alexgibson) [inline media/css/base/mozid-modal.less]: Good suggestion :+1:

**Comment by alexgibson** [comment]
Added `line-height: 1.3;` as suggested, merging just as soon as travis is green. Thanks @jpetto!

**Pull request merged**